package.json

```diff
@@ -1,26 +1,23 @@
 {
-    "name": "open-discussions-root",
+    "name": "root",
     "private": true,
     "workspaces": [
         "frontends/*"
     ],
     "scripts": {
-        "foreach-with-fallback": "yarn workspaces foreach --verbose --topological exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js $0 $1",
-        "start-dev": "yarn workspaces foreach --verbose --parallel --interlaced run start-dev",
+        "start-dev": "yarn workspaces foreach -pvi run start-dev",
         "build": "yarn workspaces foreach run build",
-        "clean": "run foreach-with-fallback clean global:clean",
```

Comment:
We weren't using `clean`, so I got rid of it.

Previously, we were running yarn commands via `run foreach-with-fallback <cmd> <fallback-cmd>`, the idea being that workspaces could define their own `cmd`, and default to a globally-defined `fallback-cmd`.

In https://github.com/mitodl/open-discussions this was useful because we were supporting differing linting/typechecking for old Flow+JS code than for new TS code.

It was a little complicated and is unnecessary now that everything is TS: all projects use the same linting, typechecking, etc commands.